ReleaseNotes.md

```diff
@@ -1,5 +1,36 @@
 # Release Notes
 
+## New in Release 2.8.0
+
+Post-training Quantization:
+
+- Breaking changes:
+  - `nncf.quantize` signature has been changed to add `mode: Optional[nncf.QuantizationMode] = None` as its 3-rd argument, between the original `calibration_dataset` and `preset` arguments.
+  - (Common) `nncf.common.quantization.structs.QuantizationMode` has been renamed to `nncf.common.quantization.structs.QuantizationScheme`
+- General:
+  - (OpenVINO) Changed default OpenVINO opset from 9 to 13.
+- Features:
```

Comment:
Missed #123999. You can add it like:
Post-training Quantization with Accuracy Control, nncf. quantize_with_accuracy_control(), has been extended by ‘restore_mode’ optional parameter to revert weights to int8 instead  of the original precision. This parameter helps to reduce the size of the quantized model and improves its performance. By default, it’s disabled and model weights are reverted to the original precision in nncf.quantize_with_accuracy_control().

Reply:
Done